types/ember/test/computed.ts

```diff
@@ -32,6 +32,15 @@ const Person = Ember.Object.extend({
         get() {
             return this.get('fullName');
         }
+    }),
+
+    fullNameSetOnly: Ember.computed('firstName', 'lastName', {
+        set(key, value) {
+            let [first, last] = value.split(' ');
+            this.set('firstName', first);
+            this.ste('lastName', last);
```

Comment:
Typo here, otherwise LGTM

Reply:
@dwickern Fixed. I had just copied it from the earlier declaration.

Reply:
Figures, that's why we need type safety!!